### PR TITLE
V2.13

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/admin/integration/spark.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+./bin/docker-image-tool.sh -t <image_name> build
+```
+
+An image named `<image_name` will be created.
+
+Then load it into your kind cluster:
+
+```
+kind load docker-image spark:<image_name> --name \<CLUSTER\_NAME\>
+```
```
Comment (enoodle) [inline docs/admin/integration/spark.md:45]:
```suggestion
Then push the docker image to your repository:

```
docker tag spark:<image_tag> <MY_REPO>/spark:<image_tag>
docker push <MY_REPO>/spark:<image_tag>
```
```

File: docs/admin/integration/spark.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+We need to build docker images and push them to either a public repository or load them to kind.
+
+To build the images run:
+
+```
+./bin/docker-image-tool.sh -t <image_name> build
+```
+
+An image named `<image_name` will be created.
```
Comment (enoodle) [inline docs/admin/integration/spark.md:39]:
```suggestion
./bin/docker-image-tool.sh -t <image_tag> build
```

An image named `spark:<image_tag>` will be created.
```

File: docs/admin/integration/spark.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+kind: Pod
+
+name: spark-pi-f4445788619bd784-driver
+
+uid: 951d5bdb-cea8-41e8-998b-9c6fb6dff846
+
+![Shape1](RackMultipart20230620-1-fcrkwu_html_932f45e5bf08d98e.gif)
+```
+
+The driver has a set amount of jobs to distribute, and will use any available executors to finish them. So, if only two executors are running and 6 jobs are available each one will execute 3 jobs.
```
Comment (enoodle) [inline docs/admin/integration/spark.md:132]: lets remove this part, it was only meant for internal use to understand how to identify spark pods.

**Review by enoodle — COMMENTED** [review]
I have added a few suggestions

File: docs/admin/integration/spark.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/admin/integration/spark.md:45]: Did you want me to add the 
```
docker tag spark:<image_tag> <MY_REPO>/spark:<image_tag>
docker push <MY_REPO>/spark:<image_tag>
```
to the file? If so then where?

File: docs/admin/integration/spark.md (hunk 3 above)
Comment (jasonnovichRunAI) [reply docs/admin/integration/spark.md:132]: ok.

**Comment by jasonnovichRunAI** [comment]
@enoodle Thank you! I have taken the suggestions and made the changes, and there was one where I was not sure, so please comment on that one. Also, I left some HTML comments for you ( They look like this  so if you could review those and answer the questions / make suggestions / changes accordingly that would be very helpful.

**Pull request merged**